Source/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -154,21 +164,17 @@ - (void)dismissExternalUserAgentAnimated:(BOOL)animated completion:(void (^)(voi
   
   [self cleanUp];
   
-  if (@available(iOS 12.0, *)) {
```

Comment:
I like the refactor of this whole section here.

Reply:
Thanks. It actually wasn't just a refactor, but to prevent problems if the user switch on/off guided access in the middle of the login process.

Reply:
Oh, not saying it wasn't needed - I just like this improvement even without the guided access fixes, I think it's the better way to structure it generally.